### PR TITLE
Update UserGuide on search and updated personal MD

### PR DESCRIPTION
**Pull request by Leeyp**

### CODE REVIEW

**Review by Vinci-Hu — APPROVED** [review]
LGTM

File: docs/UserGuide.md [hunk 1]
```diff
@@ -29,6 +29,29 @@ Example of usage:
 
 `todo n/Refactor the User Guide to remove passive voice d/13/04/2020`
 
+### Search: `search`
+Checks if a certain food item is inside the fridge, and if the item is found, informs user of the food's storage location.
+
+Format: `search FOOD_NAME`
+
+* The `FOOD_NAME` can be the name of a food but not an empty description.
+* The command requires the user to input the exact name of the food item `FOOD_NAME` to be searched for.
```
Comment (kwokyto) [inline docs/UserGuide.md:38]: i think we can use partial food names too, no need to be exact

Comment (kwokyto) [reply docs/UserGuide.md:38]: i might be wrong

**Review by kwokyto — APPROVED** [review]
LGTM

File: docs/UserGuide.md (hunk 1 above)
Comment (Leeyp) [reply docs/UserGuide.md:38]: This is unfortunately not the current implementation

**Review by joohwan58 — APPROVED** [review]
LGTM

File: docs/UserGuide.md (hunk 1 above)
Comment (SimJJ96) [reply docs/UserGuide.md:38]: yes actually any name will do as long as it is not empty

**Review by SimJJ96 — APPROVED** [review]
LGTM

**Pull request merged**